examples/sorting/Insertion Sort/Go/main.go

```diff
@@ -0,0 +1,39 @@
+// Insertion sort
+//
+// Time complexity: O(N^2)
+//   - outperforms more complex sorts with small lists due to its low hidden constant value
+//
+// Insertion sort is an in-place sort that performs comparisons to sort an array.
+// It starts at a given item in the list, and then looks to move it to the left so that
+// the sublist to the left of the chosen index is sorted in ascending order.
+//
+// This means that by the time the sort moves the last item in the array, the array
+// would already be sorted
+package main
+
+import "fmt"
+
+func insertionSort(arr []int) []int {
```

Comment:
This seems to be an 8-space tab. Can we narrow this down to be consistent with the rest.